homework/index.js

```diff
@@ -0,0 +1,36 @@
+const status = (response) => {
+  if (response.status >= 200 && response.status < 300) {
+    return Promise.resolve(response);
+  }
+  return Promise.reject(new Error(response.statusText));
+};
+
+const json = response => response.json();
+
+// fetching api and onload when page load
+window.onload = () => fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
```

Comment:
Very sleek, like the inline functions, and those chained promises (thens) immediately calling the next handler, implicitly passing the argument... Classy!